changelog.md

```diff
@@ -26,6 +26,8 @@
 ### Fixes
 
 - [#4184](https://github.com/ignite/cli/pull/4184) Set custom `InitChainer` because of manually registered modules
+- [#4198](https://github.com/ignite/cli/pull/4198) Set correct prefix overwriting in `buf.gen.pulsar.yaml`
```

Comment:
this will be added in https://github.com/ignite/cli/pull/4200/files, so not editing this and assuming we'll merge the other one first.